packages/ecs-agent/0002-bottlerocket-version-values-settable-with-linker.patch

```diff
@@ -1,30 +1,30 @@
-From 648041259821e73e79965f3de3864109c3a5a816 Mon Sep 17 00:00:00 2001
+From af86a52ad8de5ed133f30d9f09937ffd08264d6f Mon Sep 17 00:00:00 2001
 From: Samuel Karp <skarp@amazon.com>
 Date: Thu, 16 Jul 2020 10:52:01 -0700
-Subject: [PATCH] bottlerocket: version values settable with linker
+Subject: [PATCH 2/2] bottlerocket: version values settable with linker
 
 ---
  agent/version/version.go | 6 +++---
  1 file changed, 3 insertions(+), 3 deletions(-)
 
 diff --git a/agent/version/version.go b/agent/version/version.go
-index 4417a7cf..6fcb4832 100644
+index 42ea50df..80c7b249 100644
 --- a/agent/version/version.go
 +++ b/agent/version/version.go
 @@ -22,10 +22,10 @@ package version
  // repository. Only the 'Version' const should change in checked-in source code
  
  // Version is the version of the Agent
--const Version = "1.41.0"
-+var Version = "1.41.0"
+-const Version = "1.43.0"
++var Version = ""
```

Comment:
Neat, what happened so that we don't need this anymore?

Reply:
We never actually needed the value in the patch before, since we're overriding all the values via `-ldflags`.  I had previously left the version in because I had assumed it would make it easier to refresh the patch going forward, but it just caused conflicts; I'm expecting that this patch is _still_ going to cause conflicts but at least it's more clear that we don't need to preserve the value in the patch itself.